tests/acceptance/tests/basic_dsc_resources/waitforall/negative/waitforall_blacklist.rb

```diff
@@ -18,7 +18,7 @@
 dsc_manifest = ERB.new(File.read(dsc_manifest_template_path), 0, '>').result(binding)
 
 # Verify
-error_msg = /Error:.*Invalid resource type dsc_waitforall/
+error_msg = /Error:.*resource.*dsc_waitforall/
```

Comment:
I don't see "resource" being part of too many words that would fall into the context of a false positive here.